hack/e2e.go

```diff
@@ -578,6 +571,131 @@ func (k kops) Down() error {
 	return finishRunning("kops delete", exec.Command(k.path, "delete", "cluster", k.cluster, "--yes"))
 }
 
+type KubernetesAnywhere struct {
+	Path           string
```

Comment:
As discussed offline, some of these need to be public to be used in the config template, but it looks like Path can be unexported.